{"name": "Ошибка 404 Not Found при запросе на получение активных/завершенных заказов курьера с несуществующим ID курьера.", "status": "passed", "description": "Отправляем GET запрос на ручку /api/v1/orders?courierId=IDКурьера с несуществующим ID. Проверяем что ответ 404 Not Found, и сообщение в теле ответа \"Курьер с идентификатором {courierId} не найден\"", "steps": [{"name": "Генерация рандомного ID", "status": "passed", "start": 1738945780800, "stop": 1738945780800}, {"name": "GET request.", "status": "passed", "parameters": [{"name": "path", "value": "'/api/v1/orders/?courierId=908533447'"}], "start": 1738945780800, "stop": 1738945780857}], "start": 1738945780800, "stop": 1738945780857, "uuid": "9e67adbd-a73b-4afe-a111-7c750bb6db88", "historyId": "b557bd0f011fa42f897d59c1807c7d43", "testCaseId": "b557bd0f011fa42f897d59c1807c7d43", "fullName": "tests.test_get_orders_list.TestGetOrdersList#test_get_orders_list_orders_accepted_by_courier_not_valid_courier_id_shows_error_404", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_orders_list"}, {"name": "subSuite", "value": "TestGetOrdersList"}, {"name": "host", "value": "k-julls"}, {"name": "thread", "value": "14568-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_orders_list"}]}